classes/auto-package-libs.oeclass

```diff
@@ -153,14 +153,15 @@ def auto_package_libs (d):
         files += get_extra_files(devpkg)
         d.set("FILES_" + devpkg, " ".join(files))
 
+        library_version = (d.get("LIBRARY_VERSION_" + pkg) or
+                           d.get("LIBRARY_VERSION"))
+
         pkg_provides = (d.get("PROVIDES_" + pkg) or "").split()
         pkg_provides.append("%s%s"%(provideprefix,
                                     lib.replace("_", "-").lower()))
-        library_version = (d.get("LIBRARY_VERSION_" + pkg) or
-                           d.get("LIBRARY_VERSION"))
         if library_version:
-            pkg_provides.append(provideprefix + lib.replace("_","-").lower()
-                                + '-' + library_version)
+            pkg_provides.append("%s%s-%s"%(
+                provideprefix, lib.replace("_","-").lower(), library_version))
```

Comment:
This also seems to be a no-op change. I don't mind the replacement, but I'd like to make sure I understand whether this hunk is part of the functional change.

Reply:
You are right, just a no-op.  I thought it would be more clear to use same syntax style as the line above.